src/main/catalog-pusher.ts

```diff
@@ -14,8 +14,8 @@ export class CatalogPusher {
   init() {
     const disposers: Disposer[] = [];
 
-    disposers.push(reaction(() => this.catalog.items, (items) => {
-      broadcastMessage("catalog:items", toJS(items, { recurseEverything: true }));
+    disposers.push(reaction(() => toJS(this.catalog.items, { recurseEverything: true }), (items) => {
```

Comment:
Otherwise changes to catalog items won't trigger reaction.

Reply:
Okay... this breaks my mental model of how reactions work.